Assets/BossRoom/Prefabs/Game/StaticNetworkObjects.prefab

```diff
@@ -39,6 +39,9 @@ Transform:
   - {fileID: 3359597238691122721}
```

Comment:
Not for this file in particular but a comment on BossRoom scene.

It seems the BossRoom scene didn't need to be marked as changed if only the `Dungeon` and `StaticNetworkObjects` prefabs were modified. Requesting that the BossRoom scene be reverted. Followed up offline with more details.

Reply:
The scene had scene-specific overrides to the Dungeon prefab, so the scene had to be modified to lock in the changes and let the prefabs be authoritative.

Reply:
Actually, the scene still had overrides in it, in this case for `StaticNetworkObjects`. I just saved those changes to the prefab, and checked in the scene again. So now the scene should have no overrides for the major prefabs, which will let changes happen to just the prefabs going forward. (Barring anything that needs navmesh regen, at least.)

Reply:
Great. Thanks for the clarification.